app/subscriber.py

```diff
@@ -58,7 +58,28 @@ def receipt_to_case(message: Message):
     log = log.bind(questionnaire_id=questionnaire_id, created=time_obj_created, tx_id=tx_id, case_id=case_id)
 
     metadata['response_datetime'] = time_obj_created
-    send_message_to_rabbitmq(json.dumps(metadata))  # NB: in the future this should hold `questionnaire_id`
+
+    receiptMessage = {}
+    eventObject = {}
+    payloadObject = {}
+    responseObject = {}
```

Comment:
Names should be snake case and these dictionaries could all be literal definitions e.g.
```python
    event_object = {'type': 'RESPONSE_RECEIVED',
                    'source': 'RECEIPT_SERVICE',
                    'channel': 'EQ',
                    'dateTime': time_obj_created,
                    'transactionId': tx_id}
```

Reply:
done